src/player_display.cpp

```diff
@@ -189,26 +189,26 @@ static std::string dodge_skill_text( double mod )
     return string_format( _( "Dodge skill %+.1f. " ), mod );
 }
 
-static int get_encumbrance( const player &p, body_part bp, bool combine )
+static int get_player_encumbrance( const player *p, body_part bp, bool combine )
```

Comment:
Why change this to take a pointer? It should not be called with a nullptr, so why allow it?